dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd

```diff
@@ -52,17 +52,17 @@ Service-Component: \
 # for the exported API.
 Export-Package: \
     com.ibm.ws.webcontainer.speclevel, \
-    io.openliberty.servlet.internal.*factory*;provide:=true
+    io.openliberty.webcontainer.servlet.internal.*factory*;provide:=true
 
 Import-Package: \
-    !io.openliberty.servlet.internal.async.factory, \
-    !io.openliberty.servlet.internal.osgi.container.config.factory, \
-    !io.openliberty.servlet.internal.osgi.response.factory, \
-    !io.openliberty.servlet.internal.osgi.srt.factory, \
-    !io.openliberty.servlet.internal.osgi.webapp.factory, \
-    !io.openliberty.servlet.internal.servlet.factory, \
-    !io.openliberty.servlet.internal.session.impl.factory, \
-    !io.openliberty.servlet.internal.util.factory, \
+    !io.openliberty.webcontainer.servlet.internal.async.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.container.config.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.response.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.srt.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.webapp.factory, \
+    !io.openliberty.webcontainer.servlet.internal.servlet.factory, \
+    !io.openliberty.webcontainer.servlet.internal.session.impl.factory, \
+    !io.openliberty.webcontainer.servlet.internal.util.factory, \
```

Comment:
Can't comment any lower, but could you fix the indent for `com.ibm.ws.kernel.boot.core;version=latest,\` below?